c10/core/SymInt.h

```diff
@@ -27,14 +27,23 @@ namespace c10 {
 // SymInt will be extenteded to represent a union structure Union[int64_t,
 // SymIntNodeImpl*] which will be implemented as a single packed int64_t field
 // named data_.
+
+#ifdef C10_MOBILE
+#define SKIP_IS_SYMBOLIC_ON_MOBILE(_) \
+  do {                                \
+  } while (0)
+#else
+#define SKIP_IS_SYMBOLIC_ON_MOBILE(X) TORCH_CHECK(X)
```

Comment:
is this really necessary lol. You could have just #ifdef'd the use sites

Reply:
lmao, this looked more streamlined tbh, but let you know if you feel strongly about using the ifdef instead, I'll change it to ifdef.